src/entity/Human.php

```diff
@@ -365,7 +365,7 @@ public function applyDamageModifiers(EntityDamageEvent $source) : void{
 	protected function applyPostDamageEffects(EntityDamageEvent $source) : void{
 		parent::applyPostDamageEffects($source);
 		$totemModifier = $source->getModifier(EntityDamageEvent::MODIFIER_TOTEM);
-		if($totemModifier < 0){ //Totem prevented death
+		if($totemModifier <= -1){ //Totem prevented death
```

Comment:
This appears to be unnecessary.

Reply:
Yeah, since it if is less than 0 it would still mean the totem activated